sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Microsoft.Azure.Management.HealthcareApis.csproj

```
@@ -8,12 +8,12 @@
     <Description>Provides management capabilities for Microsoft Azure Healthcare Apis.</Description>
     <AssemblyTitle>Microsoft Azure Healthcare Apis Management</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Management.HealthcareApis</AssemblyName>
-    <Version>1.0.2</Version>
+    <Version>1.0.3</Version>
```

I recommend incrementing the minor version for new features.
```suggestion
    <Version>1.1.0</Version>
```